contracts/Co2knList.sol

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Lets not couple this to CO2Tkns.  I think CO2TKNs will go in another project. Call it BurnableToken whitelist. This pob-contract code adn the gelt engine are completely decoupled from our domain logic.  No carbon credit stuff goes here.

Reply:
Think of it as a general purpose PoB engine. The methodology contracts will use this project as a dependency.